### PR TITLE
fix for removal of empty pqmr dirs and some misc logline fixes

### PR DESCRIPTION
**Pull request by nkunal**
# Description
Summarize the change.

Fixes #<issue-number> (link all the GitHub issues this addresses)

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ ] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: pkg/segment/writer/segstore.go [hunk 1]
```diff
@@ -684,39 +684,38 @@ func (segstore *SegStore) AppendWipToSegfile(streamid string, forceRotate bool,
...
 		if err != nil {
-			log.Errorf("Error deleting Pqmr directory at %v", pqmrDirectory)
+			log.Errorf("removePqmrFilesAndDirectory: Error deleting Pqmr directory: %v, err: %v",
+				pqmrDirectory, err)
 			return err
 		}
-		pqmrParentDirectory := workingDirectory + "/" + fmt.Sprintf("%v/", segKey)
+		pqmrParentDirectory := fmt.Sprintf("%v/", segKey)
 		files, err = os.ReadDir(pqmrParentDirectory)
 		if err != nil {
```
[the remark this answers is not in the document]
Comment (AndrewHess) [reply pkg/segment/writer/segstore.go:710]: Lindy's comment is fine as a nit, or it can be ignored

File: pkg/segment/writer/segwriter.go [hunk 2]
```diff
@@ -401,11 +401,11 @@ func rotateSegmentOnTime() {
...
 				log.Errorf("rotateSegmentOnTime: failed to append,  streamid=%s err=%v", err, streamid)
 			} else {
+				// remove unused segstores if its has been twice
+				// the segrotation time since we last updated it
 				if time.Since(segstore.lastUpdated) > segRotateDuration*2 && segstore.RecordCount == 0 {
-					log.Infof("Deleting the segstore for streamid=%s and table=%s", streamid, segstore.VirtualTableName)
+					log.Infof("Deleting unused segstore for segkey: %v", segstore.SegmentKey)
 					delete(allSegStores, streamid)
-				} else {
-					log.Infof("Rotating segment due to time. streamid=%s and table=%s", streamid, segstore.VirtualTableName)
```
Comment (AndrewHess) [inline pkg/segment/writer/segwriter.go:408]: You deleted this log because it's not actually correct, right?

**Review by AndrewHess — COMMENTED** [review]
Looking good

File: pkg/segment/writer/segwriter.go (hunk 2 above)
Comment (nkunal) [reply pkg/segment/writer/segwriter.go:408]: yup

File: pkg/segment/writer/segstore.go [hunk 3]
```diff
@@ -684,39 +684,38 @@ func (segstore *SegStore) AppendWipToSegfile(streamid string, forceRotate bool,
...
 func removePqmrFilesAndDirectory(pqid string, segKey string) error {
-	workingDirectory, err := os.Getwd()
-	if err != nil {
-		log.Errorf("Error fetching current workingDirectory")
-		return err
-	}
-	pqFname := workingDirectory + "/" + fmt.Sprintf("%v/pqmr/%v.pqmr", segKey, pqid)
-	err = os.Remove(pqFname)
+
+	pqFname := fmt.Sprintf("%v/pqmr/%v.pqmr", segKey, pqid)
```
[the remark this answers is not in the document]
Comment (nkunal) [reply pkg/segment/writer/segstore.go]: fixed

**Review by AndrewHess — APPROVED** [review]

**Pull request merged**